backend/gis_module.py

```diff
@@ -0,0 +1,47 @@
+# Module for common GIS utilities relevant to shoulder2shoulder.
+from geopy.geocoders import Nominatim
+from geopy import distance
+
+def geocode(address):
+    '''
+    Geocode an address as a string and returns dictionarity with the standardized address and a 
+    returns a 'coords' key with a tuple that has latitude, then longitude.
+    '''
+
+    # Nominatim has a rate limit of 1 second. For testing purposes, 
+    # I am using my (Ethan's) email as the registered user.
+    geolocator = Nominatim(user_agent = "ethanarsht@gmail.com")
+    location = geolocator.geocode(address)
```

Comment:
Also, this breaks if the address doesn't return an address; maybe check if location is none before trying to extract the address information